### PR TITLE
Null to Text Bug

### PR DESCRIPTION
**Pull request by Charlie-Henry**
Error from Prefect:`InvalidJSONError: Out of range float values are not JSON compliant`

I believe the source of this is the inclusion of some pool entry passes in the Smartfolio data. They don't have a `end_time` or a `duration_min` field so these become null and breaks things. Strange part is that these rows have been included for months and only now are causing issues. 

This is only affecting the Parking Transactions dataset which is supposed to be only parking anyway. Potentially we could make a pool pass dataset as well. 

I'll have to go back and remove these rows that were imported from Socrata 👎 

I should also note that this change won't affect Finance's ability to match transactions as that is covered by a separate script. This is only for the parking transactions dataset.

Affecting Kiosk IDs:

81002119
81002120
81002121
81002122
81002123
81002124

### CODE REVIEW

**Comment by Charlie-Henry** [comment]
OK, changed things again. I was originally calculating a `duration_min` field if it was null. I think the best course is to just remove those rows. Almost all of these are pool passes and might include very few malformed parking transactions.

**Comment by Charlie-Henry** [comment]
Another commit, sorry!!

We were getting some timeout errors from Postgrest. It looks like it was caused by me defaulting to the `start_date` for searching through parking data for matches was January 1st 2021. This ends up being a huge request, so I've changed this only look at the last thirty days.

**Comment by chiaberry** [comment]
I saw the 504 on postgrest and was wondering what was up -- good to know you found a way to avoid the super huge request

File: meters/smartfolio_s3.py [hunk 1]
```diff
@@ -217,22 +217,8 @@ def transform(smartfolio):
...
-    smartfolio["timedelta"] = (
-        smartfolio["end_datetime"] - smartfolio["start_datetime"]
-    ).dt.total_seconds()
-
-    smartfolio["timedelta"] = smartfolio["timedelta"] / 60
-    smartfolio.loc[smartfolio["duration_min"].isna(), "duration_min"] = smartfolio.loc[
-        smartfolio["duration_min"].isna(), "timedelta"
-    ]
+    # If duration is null, it is because it is a pool entry transaction and doesn't have an end time
+    smartfolio = smartfolio[~smartfolio["duration_min"].isna()]
```
Comment (chiaberry) [inline meters/smartfolio_s3.py:221]: TIL what the ~ does in 🐼

**Review by chiaberry — APPROVED** [review]

**Pull request merged**